test/Telegram.Bot.Tests.Integ/Admin Bot/ChannelAdminBotTests.cs

```diff
@@ -131,7 +132,7 @@ public ChannelAdminBotTests(TestsFixture testsFixture, Fixture classFixture)
         {
             await _fixture.SendTestCaseNotificationAsync(FactTitles.ShouldSetChatPhoto);
 
-            using (Stream stream = System.IO.File.OpenRead(Constants.FileNames.Photos.Logo))
+            using (FileStream stream = File.OpenRead(Constants.FileNames.Photos.Logo))
             {
```

Comment:
Please keep is as `System.IO.File` to make it clear to user's eyes what class we are using. There also is a `Telegram.Bot.Types.File` class.